Makefile

```diff
@@ -14,6 +14,8 @@
 
 REPO_ROOT    := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 LEADER_ELECT := false
+#TODO (rishabh-11): Remove this version assignment when the latest g/g is vendored.
```

Comment:
What is "the latest"? This changes over time. Please specify an exact minimum number that is required.